hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/lock/OmReadOnlyLock.java

```diff
@@ -47,13 +51,16 @@ public void releaseMultiUserLock(String firstUser, String secondUser) {
   }
 
   @Override
-  public void releaseWriteLock(Resource resource, String... resources) {
+  public OMLockDetails releaseWriteLock(Resource resource,
+      String... resources) {
   // Intentionally empty
+    return null;
```

Comment:
Can we return EMPTY_DETAILS_LOCK_NOT_ACQUIRED for this and releaseReadLock?

Reply:
Updated.